auto_joiner.py

```diff
@@ -460,7 +460,7 @@ def join_meeting(meeting):
             send_button = wait_until_found("#send-message-button", 5)
             send_button.click()
             print(f'Sent message {config["join_message"]}')
-            discord_notification("Sent message", {config["join_message"]})
+            discord_notification(f'Sent message {config["join_message"]}')
```

Comment:
Doesn't discord_notification take 2 arguments?

Reply:
> Doesn't discord_notification take 2 arguments?

Can you please respond?

Reply:
Hi, I am extremely sorry! I was out on a holiday and I forgot to update my Github status.

Reply:
Yes it takes 2 arguments, once again i messed with the typo. Ill again solve this!

Reply:
Please check now! And sorry for the inconvenience faced!